cmd/server-main.go

```diff
@@ -183,8 +190,7 @@ func serverMain(ctx *cli.Context) {
 
 	// Configure server.
 	// Declare handler to avoid lint errors.
-	var handler http.Handler
-	handler, err = configureServerHandler(globalEndpoints)
+	handler, err := configureServerHandler(globalEndpoints)
```

Comment:
why do you need to change this line? Is above comment not applicable anymore?

Reply:
I don't see any linter errors, it was changed because of loadCerts() since err moved into that function. I can change it back.

Reply:
if you see few lines below `newObject, err :=` so current master has this sort of frivolous IMO.

Reply:
I explicitly added the comment to indicate `golint` error.  Later someone will fix/add the same.  Do you see any issue to left the line as is?

Reply:
No i can leave it as is, revert it back to how it was.